packages/cli/src/errors/external-secrets-provider-not-found.error.ts

```diff
@@ -1,6 +1,6 @@
-import { ApplicationError } from 'n8n-workflow';
+import { UnexpectedError } from 'n8n-workflow';
 
-export class ExternalSecretsProviderNotFoundError extends ApplicationError {
+export class ExternalSecretsProviderNotFoundError extends UnexpectedError {
```

Comment:
The provider name seems to be coming in a request to the controller. Hence that makes this a `UserError` as it can fail due to input from user

Reply:
My reasoning is that the frontend should never allow sending an external secrets provider name that does not exist.